frontend/src/History/HistoryRow.js

```diff
@@ -44,15 +44,56 @@ class HistoryRow extends Component {
       data
     } = this.props;
 
+    const { query, queryType } = data;
+
+    let searchQuery = query;
     let categories = [];
 
     if (data.categories) {
-      categories = data.categories.split(',').map((item) => {
-        return parseInt(item);
-      });
+      categories = data.categories.split(',').map((item) => parseInt(item));
+    }
+
+    const searchParams = [
+      'imdbId',
+      'tmdbId',
+      'tvdbId',
+      'traktId',
+      'rId',
+      'tvMazeId',
+      'season',
+      'episode',
+      'artist',
+      'album',
+      'label',
+      'track',
+      'year',
+      'genre',
+      'author',
```

Comment:
Can we do it without hard coding these? We already have to remember to update these in a few places on the backend that often get missed

Reply:
backend and front end - anyway to have these in a singular or twoish spots?

Reply:
Sure, I'll look at it when I have time.

Baker, lease avoid useless comments

Reply:
Haha, now we've gone to 3 extra places in UI to update them 😭

Reply:
Might be fine, I haven't looked close... Just saw more lists of parameters and head started spinning thinking of all the locations to add a new parameter